src/component/molecules/popup-menu/popup-menu.tsx

```diff
@@ -3,30 +3,47 @@ import PopupState, { bindTrigger, bindMenu } from "material-ui-popup-state";
 import { Menu } from "@mui/material";
 import { CategoriesModel } from "../../../api/api-get-categories";
 import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
-import {
-  PopupButtonCustomize, MenuItemCustomize
-} from "./popup-menu.styled";
+import { PopupButtonCustomize, MenuItemCustomize } from "./popup-menu.styled";
+import { useNavigate } from "react-router-dom";
 interface PopupMenuPropsModel {
   categoryName: string;
   categoriesObj: CategoriesModel[];
 }
 
-const PopupMenu: FC<PopupMenuPropsModel> = ({ categoryName, categoriesObj }) => {
+const PopupMenu: FC<PopupMenuPropsModel> = ({
+  categoryName,
+  categoriesObj,
+}) => {
+  const nav = useNavigate();
+  const handleNavigation = (categoryName: string): void => {
+    if (categoryName != "o nas") {
```

Comment:
Please do not use magic strings. "o nas" should be defined somewhere and we should refer to this constant.